UnitTests/Tests/TRWeatherUpdateCacheSpec.m

```diff
@@ -29,6 +29,14 @@ - (NSURL *)latestWeatherUpdateURL;
     [[NSFileManager defaultManager] removeItemAtURL:weatherUpdateURLForTesting() error:nil];
 };
 
+NSDictionary* (^weatherConditionsWithTemperature) (NSNumber*) = ^NSDictionary* (NSNumber *temp) {
```

Comment:
IMO this should looks like this:

    NSDictionary* (^weatherConditionsWithTemperature) (NSNumber *) = ^NSDictionary* (NSNumber *temp) {

Do you agree? I don't think that asterisk should be placed close to the opening bracket.